src/commands.ts

```diff
@@ -131,17 +124,35 @@ the active pane's buffer. Here's are the supported keys:
 All navigation commands support a repetition factor.
 For example, "5k" moves the cursor 5 lines up (type hi to hide).
 `
-            
-        }
-        else {
+            } else
+                help += "No help for " + args[0]
+        } else {
             help += `\x1B[1m${command.name}\x1B[0m\n`
-            help += `  ${command.help}\n`
-            help += `  Usage: ${command.usage}`
+                help += `  ${command.help}\n`
+                help += `  Usage: ${getUsage(shell.commands, command)}`
         }
     }
     shell.t.writeln(help)
 }
 
+function getUsage(commands: Map<string, Command>, command: Command) {
```

Comment:
This is one of these time when I put my engineering on hat and rule it doesn't pass my inspection. Not because of the code, I didn't review it yet and probably won't. It's an unnecessary complication and for quality's sake it's better to keep things simple. Having automatic usage string is not worth the extra twenty untested lines. I'd rather remove the usage all together and not have special logic for static user strings.